src/lib/viewers/image/MultiImageViewer.js

```diff
@@ -166,6 +171,9 @@ class MultiImageViewer extends ImageBaseViewer {
 
         // Give the browser some time to render before updating pannability
         setTimeout(this.updatePannability, ZOOM_UPDATE_PAN_DELAY);
+
+        // Set current page to previously opened page or first page
+        this.setPage(this.currentPageNumber - 1);
```

Comment:
whoops this wasn't supposed to be -1! I was messing with it before while testing